src/v3/src/util/deviceFingerprintingUtils.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+const getUserAgent = (): string => navigator.userAgent;
+
+const isWindowsPhone = (userAgent: string): RegExpMatchArray | null => userAgent.match(/windows phone|iemobile|wpdesktop/i);
```

Comment:
Can you move this into `src/v3/src/util/browserUtils.ts`

Reply:
Do you just mean these two functions, or the entire file?

Reply:
Oh, just the functions, sorry.